src/components/asset-list/RecyclerAssetList2/core/RawRecyclerList.tsx

```diff
@@ -167,6 +167,7 @@ const RawMemoRecyclerAssetList = React.memo(function RawRecyclerAssetList({
       onLayout={onLayout}
       ref={ref as LegacyRef<RecyclerListViewRef>}
       refreshControl={disablePullDownToRefresh ? undefined : <RefreshControl />}
+      style={{ marginBottom: -100 }}
```

Comment:
@skylarbarrera do u remember why u added this

Reply:
this was some funky shit to reduce the white space at the end of the list cause I had no clue where it was coming from